packages/atauth/src/auth_cli.c

```diff
@@ -178,8 +195,7 @@ int main(int argc, char *argv[]) {
     atlogger_log(TAG, ATLOGGER_LOGGING_LEVEL_ERROR, "Failed encoding APKAM SymmetricKey to base64\n");
     goto enc_pub_key_exit;
   }
-  atclient_atkeys_set_apkam_symmetric_key_base64(&atkeys, (const char *)apkam_symmetric_key_base64,
-                                                 apkam_symmetric_key_base64_len);
+  atclient_atkeys_set_apkam_symmetric_key_base64(&atkeys, (const char *)apkam_symmetric_key_base64, apkam_symmetric_key_base64_len);
```

Comment:
This function returns an exit code. It returns 1 if internal function malloc call fails